prisma/context.ts

```diff
@@ -5,6 +5,7 @@ export const createContext = async ({ req, res }) => {
   const prisma = new PrismaClient();
   return {
     prisma,
+    user: null,
```

Comment:
Added this for demonstration purposes for the `isAuthenticated` rule in `prisma/shield.ts`

Reply:
No problem